Pages/Hotels/hotelsHome.js

```diff
@@ -0,0 +1,481 @@
+const Commands = require("../../Pages/Commands");
+const Moments = require("../../Pages/Moments");
+
+
+class HotelsHome {
+
+    commands = new Commands
+    moments = new Moments
+
+    signInButton = '//button[contains(text(), "Sign")]'
+    signInLink = '//a[contains(@data-stid, "signin")]'
+    signUpLink = '//a[contains(@data-stid, "link-header-account-signup")]'
+    feedbackLink = "//a[text()='Feedback']"
+
+
+    goingToSearchTab = "//button[@aria-label = 'Going to']" ;
+    
+    travelersLocator = 'button[data-stid=open-room-picker]';
+    
+    adultsIncrease = "//body/div[@id='app-blossom-flex-ui']/div[@id='app-layer-manager']/div[@id='app-layer-base']/div[1]/div[2]/div[1]/div[1]/section[1]/form[1]/div[1]/div[1]/div[3]/div[1]/div[2]/div[1]/div[1]/section[1]/div[1]/div[1]/div[1]/div[1]/button[2]/span[1]/*[1]"
```

Comment:
this selector isn't good at all. When we've too many dependencies, it can break any time.